.github/workflows/publish-native-assets-to-github-releases.yml

```diff
@@ -0,0 +1,32 @@
+# This workflow will run tests using node and then publish a package to GitHub Packages when a release is created
+# For more information see: https://help.github.com/actions/language-and-framework-guides/publishing-nodejs-packages
+
+name: Add native binaries to release
+
+on:
+  release:
+    types: [created]
+    
+
+jobs:
+  augment-release:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        node-version: [8.x, 10.x, 12.x]
+        os: [ubuntu-latest, macos-latest, windows-latest]
+    steps:
+    - uses: actions/checkout@v2
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node-version }}
+    - run: |
+        npm install --build-from-source
+        ./node_modules/.bin/node-pre-gyp package
+    - name: Upload native binaries for Node ${{ matrix.node-version }} for ${{ matrix.os }}
+      uses: csexton/release-asset-action@v2
+      with:
+        pattern: "build/stage/*.tar.gz"
+        github-token: ${{ secrets.GITHUB_TOKEN }}
+        release-url: ${{ github.event.release.upload_url }}
```

Comment:
Same for this one

Reply:
This one comes from the `on: release` trigger. It's not as well documented; the trigger is documented [here](https://help.github.com/en/actions/reference/events-that-trigger-workflows#release-event-release), but the individual properties like `upload_url` are documented [here](https://developer.github.com/v3/repos/releases/#response).